src/contrib/debug.ts

```diff
@@ -0,0 +1,39 @@
+import Debug from 'debug';
+import { LoggerInterface, LOG_LEVELS, L } from '../interfaces/logger';
+// import getLogLevelConstant from '../util/logging';
+import ContribLogger from './contrib_logger';
+import ContribMetrics from './contrib_metrics';
+import { MetricsInterface } from '../interfaces/metrics';
+import Metric from '../models/metric';
+import { MindfulnessOptions } from '../interfaces/options';
```

Comment:
What is Mindfulness?

Reply:
A poor name for the core class but not worth changing for this. I'll make an issue for some changes like that.